### PR TITLE
increase mongodb test image pull timeout

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: test/extended/image_ecosystem/mongodb_replica_statefulset.go [hunk 1]
```diff
@@ -62,23 +62,21 @@ var _ = g.Describe("[image_ecosystem][mongodb][Slow] openshift mongodb replicati
...
-				desc, _ = oc.Run("describe", "pods").Output()
-				ginkgolog("\n\nPods at petset failure:\n%s\n\n", desc)
-
+				desc, _ := oc.Run("describe").Args("statefulset").Output()
+				ginkgolog("\n\nStatefulset at failure:\n%s\n\n", desc)
+				desc, _ = oc.Run("describe").Args("pods").Output()
+				ginkgolog("\n\nPods at statefulset failure:\n%s\n\n", desc)
 			}
 			o.Expect(err).NotTo(o.HaveOccurred())
-			o.Expect(podNames).Should(o.HaveLen(3))
```
Comment (bparees) [inline test/extended/image_ecosystem/mongodb_replica_statefulset.go]: why aren't we expecting 3 anymore?

**Comment by bparees** [comment]
@jim-minter I take it you saw something in the logs implying we timed out on this wait?  The analysis you provided in the issue doesn't make me think this would matter, since the pods are completely failing to start (not failing to start fast enough)

File: test/extended/image_ecosystem/mongodb_replica_statefulset.go (hunk 1 above)
Comment (jim-minter) [inline test/extended/image_ecosystem/mongodb_replica_statefulset.go]: It's duplicate code - the 3 argument to WaitForPods already ensures this.

**Comment by jim-minter** [comment]
@bparees yes, I saw the timeout activate after its 2 minutes.  The log for pod 0 showed that it had actually managed to start at 1:57, so had the timeout been longer, the test would probably have worked.

**Comment by bparees** [comment]
lgtm
[testextended][extended:core(statefulset)]

**Comment by bparees** [comment]
failing test is unrelated.
[merge]

**Comment by bparees** [comment]
rubygems flake
```
2017-02-02T19:27:15.527677000Z Gem::RemoteFetcher::FetchError: Errno::ECONNRESET: Connection reset by peer - SSL_connect (https://rubygems.org/gems/rack-1.5.2.gem)
2017-02-02T19:27:15.779916000Z An error occurred while installing rack (1.5.2), and Bundler cannot continue.
2017-02-02T19:27:15.780182000Z Make sure that `gem install rack -v '1.5.2'` succeeds before bundling.
```

[merge]

**Pull request merged**